src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -206,15 +215,33 @@ private boolean areAllChildrenIgnored() {
     }
 
     /**
-     * Returns a {@link Statement}: run all non-overridden {@code @BeforeClass} methods on this class
+     * Returns a {@link Statement}: run all non-overridden {@code @Ignore} methods on this class
+     * and superclasses before executing {@code next}.
```

Comment:
Oops, this was a copy and past error. The method withIgnores do not have a parameter named state.